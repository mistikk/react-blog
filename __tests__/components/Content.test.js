import React from 'react';
import { shallow } from 'enzyme';

import Content from '../../src/components/Content';

it('renders correctly', () => {
  const wrapper = shallow(
    <Content>
      <h1>Content</h1>
    </Content>,
  );

  expect(wrapper).toMatchSnapshot();
});
