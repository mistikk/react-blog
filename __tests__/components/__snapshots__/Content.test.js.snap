exports[`test renders correctly 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div>
    <h1>
        Content
    </h1>
</div>,
  "nodes": Array [
    <div>
      <h1>
            Content
      </h1>
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Content>
        <h1>
                Content
        </h1>
</Content>,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "children": <h1>
            Content
</h1>,
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div>
          <h1>
                    Content
          </h1>
</div>,
        "_debugID": 2,
        "_renderedOutput": <div>
          <h1>
                    Content
          </h1>
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Content>
    <h1>
        Content
    </h1>
</Content>,
}
`;
